core/navigator.ts

```diff
@@ -59,7 +59,8 @@ export class Navigator {
     const result = this.get(current)?.getFirstChild(current);
     if (!result) return null;
     // If the child isn't navigable, don't traverse into it; check its peers.
-    if (!result.isNavigable()) return this.getNextSibling(result);
+    if (!this.get(result)?.isNavigable(result))
```

Comment:
Nit: please add braces since this breaks to a new line.

Ditto for lines 89-90 and 105-106 below.

Reply:
Good catch, I...do not care for that formatter behavior. Fixed!